### PR TITLE
vomnibar: quickly access items with 'alt + num'

### PR DESCRIPTION
**Pull request by T-winkle**
Quick access by adding `accesskey` attribute to each item。
Shortcuts for various browsers on various platforms：

Browser | Windows | Linux | Mac
-- | -- | -- | --
Internet Explorer | [Alt] + accesskey | N/A |  
Chrome | [Alt] + accesskey | [Alt] + accesskey | [Control] [Alt] + accesskey
Firefox | [Alt] [Shift] + accesskey | [Alt] [Shift] + accesskey | [Control] [Alt] + accesskey
Safari | [Alt] + accesskey | N/A | [Control] [Alt] + accesskey
Opera | Opera 15 or newer: [Alt] + accesskeyOpera 12.1 or older: [Shift] [Esc] + accesskey

### CODE REVIEW

**Comment by gdh1995** [comment]
Thanks.

**Comment by gdh1995** [comment]
Does Vimium C need to hide the different modifier keys and make something like <kbd>Alt+N</kbd> work on all platforms?

**Comment by T-winkle** [comment]
> Does Vimium C need to hide the different modifier keys and make something like Alt+N work on all platforms?

That's a good idea. I also want to eliminate the differences among multiple platforms, but I don't know much about TypeScript. I just made a simple modification and you can try it.

File: front/vomnibar.html [hunk 1]
```diff
@@ -59,8 +59,9 @@
 </div>
 <div id="list">
 	<script type="text/x-template" id="template">
-	<div class="item">
+	<div class="item" accesskey="{{num}}">
 		<div class="top">
+			<span>{{num}}.</span>
```
[the remark this answers is not in the document]
Comment (T-winkle) [reply front/vomnibar.html]: Yeah, You are right, it is indeed enough to adjust "Custom CSS for Vimium C UI" .
Initially, I want to display the index number when the alt key is pressed. You can try it, if you are interested.
`Vimium C` has brought me a lot of convenience, thank you.

**Pull request merged**